blocklib/picoscope/TimingMatcher.hpp

```diff
@@ -1,15 +1,17 @@
 #ifndef GR_DIGITIZERS_TIMINGMATCHER_HPP
 #define GR_DIGITIZERS_TIMINGMATCHER_HPP
+#include <gnuradio-4.0/Block.hpp>
 #include <gnuradio-4.0/Buffer.hpp>
-#include <gnuradio-4.0/Tag.hpp>
+#include <gnuradio-4.0/Message.hpp>
```

Comment:
includes should not be necessary, probably due to previous attempts at directly emitting the error from inside the matcher?